src/plugins/point_in_time_management/public/components/empty_state/empty_state.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import {
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiTitle,
+  EuiPageContent,
+  EuiSpacer,
+  EuiText,
+  EuiPageContentBody,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiButton,
+} from '@elastic/eui';
+
+export const EmptyState = () => {
+  return (
+    <>
+      <EuiPageContent
+        className="pitEmptyState"
+        grow={false}
+        style={{ minHeight: '70vh' }}

```

Comment:
nit - these style overrides should be moved to a matching `.scss` file. But why do we need to explicitly set `minHeights` at all? My hunch is that the OUI components can be configured to handle this situation without style overrides.

Reply:
This is mainly for adding vertical space in the content body. Should I be using multiple `EuiSpacer` instead?